packages/zoe/test/unitTests/contracts/loan/helpers.js

```diff
@@ -0,0 +1,177 @@
+import '../../../../exported';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@agoric/install-ses';
+import { E } from '@agoric/eventual-send';
+
+import { setup } from '../../setupBasicMints';
+import { setupZCFTest } from '../../zcf/setupZcfTest';
+
+/**
+ * @param {import("ava").ExecutionContext<unknown>} t
+ * @param {UserSeat} seat
+ * @param {Keyword} keyword
+ * @param {IssuerKit} kit
+ * @param {Amount} expected
+ * @param {string} message
+ */
+export const checkPayout = async (
+  t,
+  seat,
+  keyword,
+  kit,
+  expected,
+  message = '',
+) => {
+  const payout = await E(seat).getPayout(keyword);
+  const amount = await kit.issuer.getAmountOf(payout);
+  t.truthy(kit.amountMath.isEqual(amount, expected), message);
+  t.truthy(seat.hasExited(), message);
```

Comment:
Why should this (and `checkPayouts()`) test `hasExited()`?  Isn't it necessarily the case if a payout exists?

Reply:
Sure, but why not test? It's a nice check of our assumptions.